apps/src/templates/DialogButtons.jsx

```diff
@@ -15,6 +15,16 @@ var DialogButtons = React.createClass({
     previousLevel: React.PropTypes.bool,
     shouldPromptForHint: React.PropTypes.bool,
     tryAgain: React.PropTypes.string,
+    userId: React.PropTypes.number
+  },
+
+  getButtonMessage: function () {
+    // Used for A/B testing on the effects of the word "hint" in the
+    // "get a hint" workflow. Anonymous users and users with even IDs
+    // should get the text "See hint", whereas users with odd IDs should
+    // see "Get a block".
+    // Test initiated March 25, 2016
```

Comment:
Please add a TODO (with an owner and a deadline [data or date]) to stop the experiment and either (1) eliminate the new behavior or (2) migrate to the new behavior.